core/CMakeLists.txt

```diff
@@ -1,14 +1,12 @@
 cmake_minimum_required(VERSION 3.8.2)
 find_package(Git REQUIRED)
 
-macro(set_conf_file)
```

Comment:
In the commit message a053844a235a0f384a48250ac7d53bd543a4d8fe,
Add the Zephyr version which this macro will be deprecated

Reply:
This macro is already deprecated.